src/topology/path_connected.lean

```diff
@@ -346,6 +372,9 @@ lemma symm_continuous_family {X ι : Type*} [topological_space X] [topological_s
   continuous ↿(λ t, (γ t).symm) :=
 h.comp (continuous_id.prod_map continuous_symm)
 
+lemma continuous_symm : continuous (symm : path x y → path y x) :=
```

Comment:
Should this be a `@[continuity]` lemma?

Reply:
Done!

Reply:
I think @urkud meant that we should make a `_root_.continuous.path_symm` similar to `_root_.continuous.path_trans` and mark that as a `continuity` lemma instead?

Reply:
Oh, I do not know: would it be possible for @urkud to perform the required modifications that I could then commit?